### PR TITLE
Changes to the Windows Batches

### PR DESCRIPTION
**Pull request by danielsdian**
Removed the requirement for virtualenv in the installation batches and added more automatic downloads

### CODE REVIEW

**Comment by k4n30** [comment]
@danielsdian going to put these tags on until your PR is ready, please advise when it's ready to go

**Comment by danielsdian** [comment]
I think I made this work. I got very confused regarding branches and forks. :(

File: windows_bat/PokemonGo-Bot-Install.bat [hunk 1]
```diff
@@ -203,14 +201,9 @@ if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
 if exist %PGBotPath% rmdir %PGBotPath% /s /q
 if not exist %PGBotPath% md %PGBotPath%
 cd C:\Python27\
-pip2 install --upgrade virtualenv
```
Comment (k4n30) [inline windows_bat/PokemonGo-Bot-Install.bat]: Should this not be presented as an option to the user whether they want virtualenv or not?

Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: I removed because after testing in some Windows machines it would cause issues in some of them. Pip just wouldnt be able to get setuptools from the online repositories, so it stuck on a loop and the bot would never start. Removing virtualenv made the bot work, but makes the user unable to run multiple bots simultaneously. I think this is fine, as these batches are intended to be used by people with very low knowledge of how to configure the bot, and running multiple bots require a bit more understanding of how it Works.

Comment (k4n30) [inline windows_bat/PokemonGo-Bot-Install.bat]: Agree with what you're saying, but you shouldn't make the decision for someone, you should let them make a decision and give them the option

File: windows_bat/PokemonGo-Bot-Install.bat [hunk 2]
```diff
@@ -203,14 +201,9 @@ if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
 if exist %PGBotPath% rmdir %PGBotPath% /s /q
 if not exist %PGBotPath% md %PGBotPath%
 cd C:\Python27\
-pip2 install --upgrade virtualenv
+pip2 install --upgrade pip
 git clone --recursive -b master https://github.com/PokemonGoF/PokemonGo-Bot %PGBotPath%
-if "%OS%" == "32-BIT" pip2 install --upgrade %PGBotPath%\windows_bat\PyYAML-3.11-cp27-cp27m-win32.whl
-if "%OS%" == "64-BIT" pip2 install --upgrade %PGBotPath%\windows_bat\PyYAML-3.11-cp27-cp27m-win_amd64.whl
```
Comment (k4n30) [inline windows_bat/PokemonGo-Bot-Install.bat]: Why did you remove these 2?

File: windows_bat/PokemonGo-Bot-Install.bat (hunk 1 above)
Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: If I leave virtualenv in the install and start batch, they will just stop in 3 out of 4 Windows machines, show a `Installing pip, setuptools` message and stay there for hours, not showing any error messages.

File: windows_bat/PokemonGo-Bot-Install.bat (hunk 2 above)
Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: PyYAML is installed by the command pip2 install -r %PGBotPath%\requirements.txt in line 206. It was redundant.

File: windows_bat/PokemonGo-Bot-Install.bat [hunk 3]
```diff
@@ -203,14 +201,9 @@ if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
...
 cd C:\Python27\
-pip2 install --upgrade virtualenv
+pip2 install --upgrade pip
 git clone --recursive -b master https://github.com/PokemonGoF/PokemonGo-Bot %PGBotPath%
-if "%OS%" == "32-BIT" pip2 install --upgrade %PGBotPath%\windows_bat\PyYAML-3.11-cp27-cp27m-win32.whl
-if "%OS%" == "64-BIT" pip2 install --upgrade %PGBotPath%\windows_bat\PyYAML-3.11-cp27-cp27m-win_amd64.whl
-cd %PGBotPath%
-virtualenv .
-call "%PGBotPath%\Scripts\activate.bat"
-pip2 install --upgrade -r %PGBotPath%\requirements.txt
```
Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: It was being done twice. Now its done only once by pip2 install -r %PGBotPath%\requirements.txt on line 206

File: windows_bat/PokemonGo-Bot-Install.bat (hunk 1 above)
Comment (k4n30) [inline windows_bat/PokemonGo-Bot-Install.bat]: No give the user the option "Do you want to install virtual env [Y / N]" if yes install, if no don't

File: windows_bat/PokemonGo-Bot-Install.bat (hunk 2 above)
Comment (k4n30) [inline windows_bat/PokemonGo-Bot-Install.bat]: https://github.com/PokemonGoF/PokemonGo-Bot/blob/dev/requirements.txt

Where?

Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: Line 13, pyyaml==3.11

Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: Protobuf is there too, in line 5: protobuf==3.0.0b4. It was removed in the last PR too. I just didnt notice the PYYAML there. I saw it last night after reading the install results and seeing the requirement fulfilled messages.

File: windows_bat/PokemonGo-Bot-Install.bat (hunk 1 above)
Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: I need to find a way to do this, as it would require changes to both install and start batchs.

Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: I added it back to the installation as it installs fine, just starting it causes the freezing. Im not gonna add it to the start batch as the start batch doesnt support multiple bot anyway.

**Comment by danielsdian** [comment]
I think this is good to merge.

File: windows_bat/PokemonGo-Bot-Install.bat (hunk 3 above)
Comment (k4n30) [inline windows_bat/PokemonGo-Bot-Install.bat]: it doesn't include the --upgrade tag though, which imho is required, as there will be some users updating, and some users installing from fresh. Needs to account for both. There is no harm in having --upgrade if the pckage doesnt exist already, but if an older version does exist, not having this flag will not update users and will cause issues

Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: Adding a --upgrade works fine until it reaches the xxhash package, where it freezes just like it happens with setuptools package. This seems to be a loop where pip fails to upgrade the package and keeps trying, for hours if you let it. Thats why I removed it.

Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: Installation of setuptools and xxhash works fine, just upgrading it wont work. Virtualenv is broken on some systems because it tries to update setuptools at start too.

Comment (danielsdian) [inline windows_bat/PokemonGo-Bot-Install.bat]: Im gonna keep trying to run it using --upgrade on my machine and some others, as soon it Works again in most of them, I can create another PR and add that function back. Right now, this issue blocks the installation to move forward.

Comment (k4n30) [inline windows_bat/PokemonGo-Bot-Install.bat]: Going to allow second review to see if it's ready for merge or not

**Comment by LuckyMe4Evers** [comment]
@danielsdian I did some research with your fixed bat and changed some things. Will show you the result.

What follows is the result of your bat file.
C:\Python27>git clone --recursive -b master https://github.com/PokemonGoF/PokemonGo-Bot c:/Test/\PokemonGo-Bot
fatal: destination path 'c:/Test/\PokemonGo-Bot' already exists and is not an empty directory.

C:\Python27>pip2 install -r c:/Test/\PokemonGo-Bot\requirements.txt
Requirement already satisfied (use --upgrade to upgrade): numpy==1.11.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 1))
Requirement already satisfied (use --upgrade to upgrade): networkx==1.11 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 2))
Obtaining pgoapi from git+https://github.com/keyphact/pgoapi.git@a2755eb42dfe49e359798d2f4defefc97fb8163d#egg=pgoapi (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 3))
  Skipping because already up-to-date.
Requirement already satisfied (use --upgrade to upgrade): geopy==1.11.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 4))
Requirement already satisfied (use --upgrade to upgrade): protobuf==3.0.0b4 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 5))
Requirement already satisfied (use --upgrade to upgrade): requests==2.10.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 6))
Requirement already satisfied (use --upgrade to upgrade): s2sphere==0.2.4 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 7))
Requirement already satisfied (use --upgrade to upgrade): gpsoauth==0.3.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 8))
Requirement already satisfied (use --upgrade to upgrade): protobuf-to-dict==0.1.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 9))
Requirement already satisfied (use --upgrade to upgrade): googlemaps==2.4.4 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 10))
Requirement already satisfied (use --upgrade to upgrade): colorama==0.3.7 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 11))
Requirement already satisfied (use --upgrade to upgrade): enum34==1.1.6 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 12))
Requirement already satisfied (use --upgrade to upgrade): pyyaml==3.11 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 13))
Requirement already satisfied (use --upgrade to upgrade): haversine==0.4.5 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 14))
Requirement already satisfied (use --upgrade to upgrade): polyline==1.3.1 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 15))
Requirement already satisfied (use --upgrade to upgrade): python-socketio==1.4.2 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 16))
Requirement already satisfied (use --upgrade to upgrade): flask==0.11.1 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): socketIO_client==0.7.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 18))
Requirement already satisfied (use --upgrade to upgrade): eventlet==0.19.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 19))
Requirement already satisfied (use --upgrade to upgrade): gpxpy==1.1.1 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 20))
Requirement already satisfied (use --upgrade to upgrade): mock==2.0.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 21))
Requirement already satisfied (use --upgrade to upgrade): timeout-decorator==0.3.2 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 22))
Requirement already satisfied (use --upgrade to upgrade): raven==5.23.0 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 23))
Requirement already satisfied (use --upgrade to upgrade): demjson==2.2.4 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 24))
Requirement already satisfied (use --upgrade to upgrade): greenlet==0.4.9 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 25))
Requirement already satisfied (use --upgrade to upgrade): yoyo-migrations==5.0.3 in c:\python27\lib\site-packages (from -r c:/Test/\PokemonGo-Bot\requirements.txt (line 26))
Requirement already satisfied (use --upgrade to upgrade): decorator>=3.4.0 in c:\python27\lib\site-packages (from networkx==1.11->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 2))
Requirement already satisfied (use --upgrade to upgrade): six in c:\python27\lib\site-packages (from pgoapi->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 3))
Requirement already satisfied (use --upgrade to upgrade): xxhash in c:\python27\lib\site-packages (from pgoapi->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 3))
Requirement already satisfied (use --upgrade to upgrade): setuptools in c:\python27\lib\site-packages (from protobuf==3.0.0b4->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 5))
Requirement already satisfied (use --upgrade to upgrade): future in c:\python27\lib\site-packages (from s2sphere==0.2.4->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 7))
Requirement already satisfied (use --upgrade to upgrade): pycryptodomex>=3.0 in c:\python27\lib\site-packages (from gpsoauth==0.3.0->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 8))
Requirement already satisfied (use --upgrade to upgrade): python-engineio>=0.8.0 in c:\python27\lib\site-packages (from python-socketio==1.4.2->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 16))
Requirement already satisfied (use --upgrade to upgrade): itsdangerous>=0.21 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): click>=2.0 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): Werkzeug>=0.7 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): Jinja2>=2.4 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): websocket-client in c:\python27\lib\site-packages (from socketIO_client==0.7.0->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 18))
Requirement already satisfied (use --upgrade to upgrade): funcsigs>=1; python_version < "3.3" in c:\python27\lib\site-packages (from mock==2.0.0->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 21))
Requirement already satisfied (use --upgrade to upgrade): pbr>=0.11 in c:\python27\lib\site-packages (from mock==2.0.0->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 21))
Requirement already satisfied (use --upgrade to upgrade): contextlib2 in c:\python27\lib\site-packages (from raven==5.23.0->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 23))
Requirement already satisfied (use --upgrade to upgrade): python-slugify in c:\python27\lib\site-packages (from yoyo-migrations==5.0.3->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 26))
Requirement already satisfied (use --upgrade to upgrade): iniherit in c:\python27\lib\site-packages (from yoyo-migrations==5.0.3->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 26))
Requirement already satisfied (use --upgrade to upgrade): MarkupSafe in c:\python27\lib\site-packages (from Jinja2>=2.4->flask==0.11.1->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): Unidecode>=0.04.16 in c:\python27\lib\site-packages (from python-slugify->yoyo-migrations==5.0.3->-r c:/Test/\PokemonGo-Bot\requirements.txt (line 26))
Installing collected packages: pgoapi
  Running setup.py develop for pgoapi
Successfully installed pgoapi-1.1.6



--------------------Restoring Backup--------------------




C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\encrypt.so COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\encrypt.so c:/Test/\PokemonGo-Bot

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\encrypt.dll COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\encrypt.dll c:/Test/\PokemonGo-Bot

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\encrypt_64.dll COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\encrypt_64.dll c:/Test/\PokemonGo-Bot

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\config.json COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\config.json c:/Test/\PokemonGo-Bot\configs\

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\userdata.js COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\\userdata.js c:/Test/\PokemonGo-Bot\web\config\

C:\Python27>if exist c:/Test/\Install-Files\encrypt.so COPY c:/Test/\Install-Files\encrypt.so c:/Test/\PokemonGo-Bot

C:\Python27>if exist c:/Test/\Install-Files\encrypt.dll COPY c:/Test/\Install-Files\encrypt.dll c:/Test/\PokemonGo-Bot

C:\Python27>if exist c:/Test/\Install-Files\encrypt_64.dll COPY c:/Test/\Install-Files\encrypt_64.dll c:/Test/\PokemonGo-Bot

C:\Python27>if exist c:/Test/\Install-Files\config.json COPY c:/Test/\Install-Files\config.json c:/Test/\PokemonGo-Bot\configs\

C:\Python27>if exist c:/Test/\Install-Files\userdata.js COPY c:/Test/\Install-Files\userdata.js c:/Test/\PokemonGo-Bot\web\config\

**What comes next is the result of my adapted file.**
Requirement already satisfied (use --upgrade to upgrade): numpy==1.11.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 1))
Requirement already satisfied (use --upgrade to upgrade): networkx==1.11 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 2))
Obtaining pgoapi from git+https://github.com/keyphact/pgoapi.git@a2755eb42dfe49e359798d2f4defefc97fb8163d#egg=pgoapi (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 3))
  Skipping because already up-to-date.
Requirement already satisfied (use --upgrade to upgrade): geopy==1.11.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 4))
Requirement already satisfied (use --upgrade to upgrade): protobuf==3.0.0b4 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 5))
Requirement already satisfied (use --upgrade to upgrade): requests==2.10.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 6))
Requirement already satisfied (use --upgrade to upgrade): s2sphere==0.2.4 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 7))
Requirement already satisfied (use --upgrade to upgrade): gpsoauth==0.3.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 8))
Requirement already satisfied (use --upgrade to upgrade): protobuf-to-dict==0.1.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 9))
Requirement already satisfied (use --upgrade to upgrade): googlemaps==2.4.4 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 10))
Requirement already satisfied (use --upgrade to upgrade): colorama==0.3.7 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 11))
Requirement already satisfied (use --upgrade to upgrade): enum34==1.1.6 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 12))
Requirement already satisfied (use --upgrade to upgrade): pyyaml==3.11 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 13))
Requirement already satisfied (use --upgrade to upgrade): haversine==0.4.5 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 14))
Requirement already satisfied (use --upgrade to upgrade): polyline==1.3.1 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 15))
Requirement already satisfied (use --upgrade to upgrade): python-socketio==1.4.2 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 16))
Requirement already satisfied (use --upgrade to upgrade): flask==0.11.1 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): socketIO_client==0.7.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 18))
Requirement already satisfied (use --upgrade to upgrade): eventlet==0.19.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 19))
Requirement already satisfied (use --upgrade to upgrade): gpxpy==1.1.1 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 20))
Requirement already satisfied (use --upgrade to upgrade): mock==2.0.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 21))
Requirement already satisfied (use --upgrade to upgrade): timeout-decorator==0.3.2 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 22))
Requirement already satisfied (use --upgrade to upgrade): raven==5.23.0 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 23))
Requirement already satisfied (use --upgrade to upgrade): demjson==2.2.4 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 24))
Requirement already satisfied (use --upgrade to upgrade): greenlet==0.4.9 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 25))
Requirement already satisfied (use --upgrade to upgrade): yoyo-migrations==5.0.3 in c:\python27\lib\site-packages (from -r c:\Test\PokemonGo-Bot\requirements.txt (line 26))
Requirement already satisfied (use --upgrade to upgrade): decorator>=3.4.0 in c:\python27\lib\site-packages (from networkx==1.11->-r c:\Test\PokemonGo-Bot\requirements.txt (line 2))
Requirement already satisfied (use --upgrade to upgrade): six in c:\python27\lib\site-packages (from pgoapi->-r c:\Test\PokemonGo-Bot\requirements.txt (line 3))
Requirement already satisfied (use --upgrade to upgrade): xxhash in c:\python27\lib\site-packages (from pgoapi->-r c:\Test\PokemonGo-Bot\requirements.txt (line 3))
Requirement already satisfied (use --upgrade to upgrade): setuptools in c:\python27\lib\site-packages (from protobuf==3.0.0b4->-r c:\Test\PokemonGo-Bot\requirements.txt (line 5))
Requirement already satisfied (use --upgrade to upgrade): future in c:\python27\lib\site-packages (from s2sphere==0.2.4->-r c:\Test\PokemonGo-Bot\requirements.txt (line 7))
Requirement already satisfied (use --upgrade to upgrade): pycryptodomex>=3.0 in c:\python27\lib\site-packages (from gpsoauth==0.3.0->-r c:\Test\PokemonGo-Bot\requirements.txt (line 8))
Requirement already satisfied (use --upgrade to upgrade): python-engineio>=0.8.0 in c:\python27\lib\site-packages (from python-socketio==1.4.2->-r c:\Test\PokemonGo-Bot\requirements.txt (line 16))
Requirement already satisfied (use --upgrade to upgrade): itsdangerous>=0.21 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:\Test\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): click>=2.0 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:\Test\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): Werkzeug>=0.7 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:\Test\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): Jinja2>=2.4 in c:\python27\lib\site-packages (from flask==0.11.1->-r c:\Test\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): websocket-client in c:\python27\lib\site-packages (from socketIO_client==0.7.0->-r c:\Test\PokemonGo-Bot\requirements.txt (line 18))
Requirement already satisfied (use --upgrade to upgrade): funcsigs>=1; python_version < "3.3" in c:\python27\lib\site-packages (from mock==2.0.0->-r c:\Test\PokemonGo-Bot\requirements.txt (line 21))
Requirement already satisfied (use --upgrade to upgrade): pbr>=0.11 in c:\python27\lib\site-packages (from mock==2.0.0->-r c:\Test\PokemonGo-Bot\requirements.txt (line 21))
Requirement already satisfied (use --upgrade to upgrade): contextlib2 in c:\python27\lib\site-packages (from raven==5.23.0->-r c:\Test\PokemonGo-Bot\requirements.txt (line 23))
Requirement already satisfied (use --upgrade to upgrade): python-slugify in c:\python27\lib\site-packages (from yoyo-migrations==5.0.3->-r c:\Test\PokemonGo-Bot\requirements.txt (line 26))
Requirement already satisfied (use --upgrade to upgrade): iniherit in c:\python27\lib\site-packages (from yoyo-migrations==5.0.3->-r c:\Test\PokemonGo-Bot\requirements.txt (line 26))
Requirement already satisfied (use --upgrade to upgrade): MarkupSafe in c:\python27\lib\site-packages (from Jinja2>=2.4->flask==0.11.1->-r c:\Test\PokemonGo-Bot\requirements.txt (line 17))
Requirement already satisfied (use --upgrade to upgrade): Unidecode>=0.04.16 in c:\python27\lib\site-packages (from python-slugify->yoyo-migrations==5.0.3->-r c:\Test\PokemonGo-Bot\requirements.txt (line 26))
Installing collected packages: pgoapi
  Running setup.py develop for pgoapi
Successfully installed pgoapi-1.1.6



--------------------Restoring Backup--------------------




C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\encrypt.so COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\encrypt.so c:\Test\PokemonGo-Bot

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\encrypt.dll COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\encrypt.dll c:\Test\PokemonGo-Bot

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\encrypt_64.dll COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\encrypt_64.dll c:\Test\PokemonGo-Bot

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\config.json COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\config.json c:\Test\PokemonGo-Bot\configs\

C:\Python27>if exist C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\userdata.js COPY C:\Users\Peter\Documents\GitHub\PokemonGo-Bot\windows_bat\userdata.js c:\Test\PokemonGo-Bot\web\config\

C:\Python27>if exist c:\Test\Install-Files\encrypt.so COPY c:\Test\Install-Files\encrypt.so c:\Test\PokemonGo-Bot

C:\Python27>if exist c:\Test\Install-Files\encrypt.dll COPY c:\Test\Install-Files\encrypt.dll c:\Test\PokemonGo-Bot

C:\Python27>if exist c:\Test\Install-Files\encrypt_64.dll COPY c:\Test\Install-Files\encrypt_64.dll c:\Test\PokemonGo-Bot

C:\Python27>if exist c:\Test\Install-Files\config.json COPY c:\Test\Install-Files\config.json c:\Test\PokemonGo-Bot\configs\

C:\Python27>if exist c:\Test\Install-Files\userdata.js COPY c:\Test\Install-Files\userdata.js c:\Test\PokemonGo-Bot\web\config\

**Comment by LuckyMe4Evers** [comment]
@danielsdian I think there's a problem with the c:/Test/ format. I will send you on slack my adapted file

**Comment by danielsdian** [comment]
I think the issue is caused by the / in your path. I removed that function we used to treat that because it caused some issues where the path would end up with three \\\ in the middle of the path. `c:/Test/\PokemonGo-Bot `
